pkg/virt-launcher/virtwrap/converter/archconverter.go

```diff
@@ -0,0 +1,49 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright the KubeVirt Authors.
+ *
+ */
+
+package converter
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+type ArchConverter interface {
+	AddGraphicsDevice(vmi *v1.VirtualMachineInstance, domain *api.Domain, c *ConverterContext)
+	SCSIController(c *ConverterContext, driver *api.ControllerDriver) api.Controller
+	isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool
+}
+
+func GetArchConverter(arch string) ArchConverter {
```

Comment:
nit: what about calling this function `NewArchConvertor`?

Reply:
I have renamed the function